src/Build/BackEnd/Components/SdkResolution/SdkLogger.cs

```diff
@@ -20,6 +21,21 @@ public SdkLogger(LoggingContext loggingContext)
             _loggingContext = loggingContext;
         }
 
+        public override void LogEvent(params object[] args)
+        {
+            MSBuildEventSource.Log.SdkResolverEvent(args);
```

Comment:
Are these actually used anywhere?

Reply:
These are new methods that allow SDK resolver authors to log events to the same stream and augment the logging built into MSBuild.  I'll be calling them in new versions of the NuGet SDK resolver so we can measure how long that one takes.  I thought it made sense to add to the MSBuild logging rather than have NuGet use some other system.  But I'm okay with leaving this out if anyone feels like its not worth having.

Reply:
Works for me!